code/.swcrc

```diff
@@ -17,7 +17,6 @@
     "target": "es2020",
     "loose": false,
     "externalHelpers": false,
-    // Requires v1.2.50 or upper and requires target to be es2016 or upper.
```

Comment:
this change is needed as I'm parsing this file as JSON in `code/jest.config.base.js`